frontend-public/src/components/LandingPage.js

```diff
@@ -78,7 +78,6 @@ export const LandingPage = () => (
             Wait for an email confirming that your employees/delegates have been added to MineSpace.
           </li>
           <li>Log in to the system.</li>
-          <li>If you require a tutorial on how to use the system, watch this video.</li>
```

Comment:
In case anyone is wondering about this deletion, Aaron asked for it to be removed until a video is created.